streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java

```diff
@@ -184,9 +184,9 @@
      * <pre>
      * KafkaStreams streams = ... // some aggregation on value type double
      * String queryableStoreName = "storeName" // the store name should be the name of the store as defined by the Materialized instance
-     * ReadOnlyKeyValueStore<K, ValueAndTimestamp<VOut>> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<K, ValueAndTimestamp<VOut>> timestampedKeyValueStore());
+     * ReadOnlyKeyValueStore&gt;K, ValueAndTimestamp&gt;VOut&lt;&lt; localStore = streams.store(queryableStoreName, QueryableStoreTypes.&gt;K, ValueAndTimestamp&gt;VOut&lt;&lt; timestampedKeyValueStore());
```

Comment:
I seem the right fix would be to add the missing `{@code }` annotation in L184?
```
<pre>{@code
```
The closing `}` is already in L190.

Reply:
Ah, good catch I'll update this and the next.